xemutest/test_nxdk_pgraph_tests.py

```diff
@@ -0,0 +1,71 @@
+"""Test harness for nxdk_pgraph_tests."""
+
+import logging
+import os
+import shutil
+
+from pyfatx import Fatx
+import test_base
+
+log = logging.getLogger(__file__)
+
+class TestNXDKPgraphTests(test_base.TestBase):
+    """Runs the nxdk_pgraph_tests suite and validates output."""
+
+    def __init__(self, test_env: test_base.TestEnvironment, results_path: str, test_data_path: str):
+        shutil.rmtree(results_path)
```

Comment:
This will fail if results path doesn't already exist. Also, the other test/base does not remove old results in the constructor. Maybe we should handle cleaning out old results in `__main__.py` generally.

Reply:
Agreed, moved and set the ignore_errors param.